src/__tests__/CompareResults/ResultsView.test.tsx

```diff
@@ -29,7 +29,8 @@ function renderWithRoute(component: ReactElement) {
 
   return renderWithRouter(component, {
     route: '/compare-results/',
-    search: '?baseRev=spam&baseRepo=mozilla-central&framework=2',
+    search:
+      '?baseRev=spam&baseRepo=mozilla-central&framework=2&platform=Windows%2COSX%2CLinux%2CAndroid&status=No+changes%2CImprovement%2CRegression&confidence=Low%2CMedium%2CHigh',
```

Comment:
it would be good that we don't have to change all these urls... that is when we use the URL without the filters, it would just use the default filters.